### ISSUE TITLE
i cant update nginxproxymanager 

### ISSUE THREAD

**Issue by arteck** [tteck/Proxmox#2706]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

i have installed it with the cmd 
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh)"

all is fine but i cant update it..

i see in the sourcecode of bash the update function, but how can I execute it?
https://github.com/tteck/Proxmox/blob/98e8955ec1db4424df229083a8d1c8d7e513ee3c/ct/nginxproxymanager.sh#L56

this is what i see 
![grafik](https://github.com/tteck/Proxmox/assets/6681528/8b03b8b7-d8bb-4e0e-a820-1d55b9dbf3c8)

i dont need a new lxc... 

how is the trick ???

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

**Comment by tteck** [comment]
🛑 Failure to comply with the following guideline.
> - For suggestions, questions or feature/script requests, please share them in the [Discussions section.](https://github.com/tteck/Proxmox/discussions)

Type the command `update` in the LXC console.

**Comment by arteck** [comment]
![grafik](https://github.com/tteck/Proxmox/assets/6681528/6c82b207-1798-42d4-b858-5298cc4fc89a)

**Comment by tteck** [comment]
🛑 Failure to comply with the following guideline.
> - Prior to submitting, kindly search the closed issues to check if the problem you are reporting has already been addressed and resolved. If you come across a closed issue that pertains to your problem, please leave a comment on that issue instead of creating a new one.

https://github.com/tteck/Proxmox/issues/2658

**Comment by arteck** [comment]
next problem
![grafik](https://github.com/tteck/Proxmox/assets/6681528/8d1bdf4b-946e-4ed5-81de-97b0326c4a36)

**Comment by arteck** [comment]
the path is wrong in your script

here

https://github.com/tteck/Proxmox/blob/98e8955ec1db4424df229083a8d1c8d7e513ee3c/ct/nginxproxymanager.sh#L139C6-L139C16

you are in /home/manager
![grafik](https://github.com/tteck/Proxmox/assets/6681528/9320a40e-b589-4661-968d-d06b8b010a3c)

but not in the new folder

**Comment by tteck** [comment]
The path is correct, your method is wrong. Please read the code.
https://github.com/tteck/Proxmox/blob/98e8955ec1db4424df229083a8d1c8d7e513ee3c/ct/nginxproxymanager.sh#L86

**Comment by arteck** [comment]
sry yes you right....

then here
![grafik](https://github.com/tteck/Proxmox/assets/6681528/61d00497-1c8b-4d43-b508-25799a7cbd20)

**Comment by tteck** [comment]
I have no clue what you're doing. I'd suggest creating a new Nginx Proxy Manager LXC.

**Comment by tteck** [comment]
The directory `/home/manager/` is not generated by the Nginx Proxy Manager LXC script available in this repository. It appears that your Nginx Proxy Manager LXC instance was set up using a different method.

**Comment by arteck** [comment]
I would like to update my nginxproxymanager
I don't want to create a new LXC

so ..here is my solution

you need
npm install pnpm@8.15.4

and i modify the frontend webpack file 
add the 
```
optimization: {
        minimize: false
    }

```
 then i can compile the frontend

and voila 
![grafik](https://github.com/tteck/Proxmox/assets/6681528/567b60ed-fa70-4183-abe8-2c796b744c47)

**Comment by Mrdindon** [comment]
I know this "issue" is closed but I ran into the same situation. Maybe my installation was too old but this is how it worked for me :

I was running nginx proxy manager v2.10.4 installed originaly from the script. Trying to update to the latest release generate an error after the script is launched :

```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh)"

...

 - Installing pnpm  /
[ERROR] in line 65: exit code 0: while executing command npm install -g pnpm@8.15 &> /dev/null
```

I ran "npm install -g pnpm@8.15" manually from the lxc console :

```

npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

added 1 package, and audited 2 packages in 1s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice 
npm notice New major version of npm available! 8.11.0 -> 10.5.1
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.5.1
npm notice Run npm install -g npm@10.5.1 to update!
npm notice 

```
Then re-ran the update script and the update went smoothly.
